app/components/login/googleAuthButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+import classNames from "classnames";
+import GoogleImg from "./googleImg";
+import ContinueMessage from "./continueMessage";
+
+export default function GoogleAuthButton() {
+    return (
+        <div
+            className={classNames(
+                "w-[100%]",
+                "content-center",
+                "flexbox m-auto",
+                "min-w-[350px]",
+                "items-center",
+                "justify-center",
+                "pt-44"
+            )}
+        >
+            <button 
+                type="button"
+                className={classNames(
+                    "rounded-full",
+                    "bg-green-900",
+                    "w-[70%]",
```

Comment:
similarly "h-full" 

Also, keep mind you can use fractional values like 'w-3/4.' Idk if there's one equivalent to 70% though